protocol/features.json

```diff
@@ -66,7 +66,14 @@
       "0056-REWA-105",
       "0056-REWA-106",
       "0056-REWA-107",
-      "0056-REWA-108"
+      "0056-REWA-108",
+      "0083-RFPR-052",
+      "0056-REWA-053",
```

Comment:
@cdummett - should these be RFPR rather than REWA?

Reply:
Yes it should! Thanks for catchng